terraform/modules/origin/security_groups.tf

```diff
@@ -15,13 +15,14 @@ resource "aws_security_group_rule" "service_from_origin_alb_http" {
   source_security_group_id = aws_security_group.origin_alb.id
 }
 
-resource "aws_security_group_rule" "origin_alb_from_any_https" {
-  description       = "${local.mode}-origin ALB allows requests from CIDRs list over HTTPS"
+# We use a secret custom header to authenticate request via the origin ALB
```

Comment:
Maybe move this comment down to the `cidr_blocks` line?

Reply:
thank. I can do that.